server/server.js

```diff
@@ -71,6 +71,20 @@ app.get(`${apiBaseUrl}/participants`,
     return res.json({ data: result });
   }));
 
+// Engage participant
+app.post(`${apiBaseUrl}/engage-participant`,
+  keycloak.allowRolesMiddleware('health_authority', 'employer'),
+  keycloak.getUserInfoMiddleware(),
+  asyncMiddleware(async (req, res) => {
+    const user = req.hcapUserInfo;
+    const result = await setParticipantStatus(
+      user.id,
+      req.body.participantId,
+      req.body.disengage ? 'open' : 'prospecting',
```

Comment:
we might want a different status here - like "archived" or something - so that in future we still have an easy way to separate out these participants into a separate tab

Reply:
Looking at the designs again I think we want "rejected here" - it's okay if there's no real filtering happening on the FE for now for that status, if we have it tracked in the DB it will be useful in the next release anyway :)